packages/react/src/themes/teams/components/Alert/alertStyles.ts

```diff
@@ -100,6 +100,16 @@ const alertStyles: ComponentSlotStylesInput<AlertProps, AlertVariables> = {
     margin: v.actionsMargin,
   }),
 
+  header: (): ICSSInJSStyle => ({
+    fontWeight: 'bold',
+    marginRight: '10px',
```

Comment:
Let's create a variable `headerMargin` and `headerFontWeight`. We can also use there `pxToRem()`:

```tsx
headerFontWeight: siteVars.fontWeightBold,
headerMargin: `0 ${pxToRem(10)} 0 0`,
```